internal/backend/grpc/validate.go

```diff
@@ -70,3 +83,81 @@ func validatePGPPublicKey(armored []byte, opts ...pgp.ValidationOption) (publicK
 		expiration: expiration,
 	}, nil
 }
+
+func (s *managementServer) ValidateJSONSchema(ctx context.Context, request *management.ValidateJsonSchemaRequest) (*management.ValidateJsonSchemaResponse, error) {
```

Comment:
It almost feels like it should have been a dry-run Update/Create which would run the validation and report errors back? 


(I don't want to block this PR, just saying)

Reply:
It reports the errors back but they're not in the machine readable format.